js/statsView.js

```diff
@@ -4,3 +4,5 @@ articleView.renderFooter = function() {
 };
 
 Project.fetchAll(articleView.renderFooter);
+
```

Comment:
Great job repeating and clarifying components of TODO's that are relevant to other parts of your code/files.